code/components/voip-mumble/include/MumbleAudioOutput.h

```diff
@@ -185,11 +180,6 @@ class MumbleAudioOutput
 			return context && context->destination();
 		}
 
-		virtual bool ShouldManagePoll()
```

Comment:
This should be kept because as said in the comment above, NuiAudioSink.cpp will handle polling for native audio so that it runs synchronously with the game mixer.